### PR TITLE
[DRAFT] Introduce luci-value-py-test

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit introduces luci-value-py-test to reduce long test duration.

Related: #12071
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/common-artifacts/CMakeLists.txt [hunk 1]
```diff
@@ -49,6 +49,7 @@ if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
     COMMAND ${CMAKE_COMMAND} -E echo "flatbuffers==23.5.26" >> ${REQUIREMENTS_OVERLAY_PATH_TF_2_12_1}
     COMMAND ${CMAKE_COMMAND} -E echo "protobuf==4.23.3" >> ${REQUIREMENTS_OVERLAY_PATH_TF_2_12_1}
     COMMAND ${CMAKE_COMMAND} -E echo "pydot==1.4.2" >> ${REQUIREMENTS_OVERLAY_PATH_TF_2_12_1}
+    COMMAND ${CMAKE_COMMAND} -E echo "pytest==7.4.3" >> ${REQUIREMENTS_OVERLAY_PATH_TF_2_12_1}
```
Comment (mhs4670go) [inline compiler/common-artifacts/CMakeLists.txt:52]: https://archlinuxarm.org/packages/aarch64/python-pytest

File: compiler/luci-value-py-test/conftest.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import re
```
Comment (mhs4670go) [inline compiler/luci-value-py-test/conftest.py:1]: This file is for pytest configuration.

https://docs.pytest.org/en/7.1.x/example/parametrize.html#parametrizing-tests

**Comment by mhs4670go** [comment]
I haven't test on the cross platform yet..

**Comment by seanshpark** [comment]
looks good to me :)

**Pull request closed without merging**